dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -118,10 +139,34 @@ public void start() {
         }
     }
 
+    /**
+     * start
+     */
+    public void start() {
+        initPlugin();
+        initRemoteServer();
+        logger.info("alert server ready start ");
+        runSender();
+    }
+
+    /**
+     * stop
+     */
+    public void stop() {
+        this.server.close();
+        logger.info("alert server shut down");
+    }
+
     public static void main(String[] args) {
         System.out.println(System.getProperty("user.dir"));
```

Comment:
Change to log output

Reply:
System.out.println is not clear who added it, so I did not move.